config/passport.js

```diff
@@ -138,6 +138,7 @@ passport.use(new FacebookStrategy({
           user.profile.name = user.profile.name || `${profile.name.givenName} ${profile.name.familyName}`;
           user.profile.gender = user.profile.gender || profile._json.gender;
           user.profile.picture = user.profile.picture || `https://graph.facebook.com/${profile.id}/picture?type=large`;
+          user.profile.location = (profile._json.location) ? profile._json.location.name : '';
```

Comment:
Missing a || condition:  If the user has set their location in, user.profile.location, the default behavior needs to not overwrite the user's location.

Reply:
> Can you provide a reference to the part of the FB dev documentation about this?

Sorry, I recently received the notification. It [here](https://developers.facebook.com/docs/graph-api/reference/user/)

> Missing a || condition: If the user has set their location in, user.profile.location, the default behavior needs to not overwrite the user's location.
Okie, I fixed